protos/sources/vega/commands/v1/transaction.proto

```diff
@@ -33,6 +33,12 @@ message InputData {
   // `block_height` prevents replay attacks in conjunction with `nonce` (see above).
   uint64 block_height = 2;
 
+  // A number of blocks this transaction is valid for starting from block_height.
+  // If block_height is set to 100, and this field is set to 10, this transaction
+  // will be rejected if it is received after block 110.
```

Comment:
I think it could be rephrased more accurately because it's not so much about when it is *received* but more that if it's not included in the blocks preceding 111, it will be rejected. In this case though it will just disappear into the void. The other case is that it's already expired when being added to the mempool which is I guess what you're describing, but just wanted to highlight that this is not the only case.